index.js

```diff
@@ -42,3 +42,229 @@ function resizeWindow() {
+    sourceBtn.classList.toggle('mobile-btn');
+    liveSiteBtn.classList.toggle('mobile-btn');
+  }
+
+  div.appendChild(projectsTitle);
+  div.appendChild(projectsLanguages);
+  div.appendChild(images);
+  div.appendChild(paragraph);
+  div.appendChild(liveSiteBtn);
+  div.appendChild(sourceBtn);
+  div.appendChild(projectExitBtn);
+  projectsSection.appendChild(div);
+
+  projectExitBtn.addEventListener('click', () => {
+    div.remove();
+    document.body.classList.toggle('no-scroll');
+    document.body.style.background = '#fff';
+    blurProjects.forEach((project) => project.classList.toggle('blur'));
+  });
+}
+
+const grid = document.getElementById('projects-grid');
+function createProjectsContainer() {
+  for (let i = 1; i < projects.length; i += 1) {
+    const projectContainer = document.createElement('div');
+    projectContainer.setAttribute('class', 'mini-projects');
+    projectContainer.setAttribute('id', `mini-project${i}`);
+    grid.appendChild(projectContainer);
+  }
+}
+
+createProjectsContainer();
+const workSectionDiv = document.querySelectorAll('.mini-projects');
+
+let counter = 0;
+function createElements(project) {
+  project.innerHTML = `
+          <img
+            src=${projects[counter].image}
+            alt="project"
+            class="hidden"
+            data-img="image"
+          />
+          <h3 class="mini-h3" data-title="title">
+          ${projects[counter].title}
+          </h3>
+          <p class="mini-p" data-p="paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="mini-languages-container" data-language="language-list">
+            <div class="mini-box"><span>html</span></div>
+            <div class="mini-box"><span>bootstrap</span></div>
+            <div class="mini-box"><span>Ruby</span></div>
+          </div>
+          <button
+            type="button"
+            class="btn project-btn"
+            id="mini-project-btn${counter + 1}"
+          >
+            See Project
+          </button>`;
+  counter += 1;
+}
+
+workSectionDiv.forEach(createElements);
+
+const buttons = [...document.querySelectorAll('.project-btn')];
+
+for (let i = 0; i < buttons.length; i += 1) {
+  buttons[i].addEventListener('click', () => {
+    createDiv(i);
+    document.body.style.background = 'black';
+    document.body.classList.toggle('no-scroll');
+    blurProjects.forEach((project) => project.classList.toggle('blur'));
+  });
+}
+
+/* Form validation */
+
+const form = document.getElementById('contact-register');
+const email = document.getElementsById('email');
+const lowerCaseRegex = /[A-Z]/;
+
+form.addEventListener('submit', (e) => {
+  if (lowerCaseRegex.test(email.value.trim())) {
+    e.preventDefault();
+    const errorMessage = document.querySelector('small');
+    errorMessage.classList.remove('hidden');
+  }
+});
```

Comment:
kindly fix the lowercase logic. is not working and it should throw a message error like `the email should be in the lowercase